tests/idris2/perror009/expected

```diff
@@ -0,0 +1,19 @@
+1/1: Building Error1 (Error1.idr)
+Error: Couldn't parse any alternatives:
+1: Expected name.
+
+Error1:4:3--4:4
+ 1 | 
+ 2 | data Foo : Type where
+ 3 |   FooBase : Foo
+ 4 |   (@) : Foo -> Foo -> Foo
```

Comment:
Is this suggestion a replacement for the addition of `isNotReservedSymbol`? I ask because with `mustWork` in that location, a failure to find a non-reserved symbol will indeed fall over right away, but we will see "Expected operator" instead of "Can't use reserved symbol @."

Reply:
I think "Can't use reserved symbol @." is a much more valuable error message so `isNotReservedSymbol` should stay. If you find further problems with the change we can just keep it as it and make a note of it

Reply:
Sounds good. I’ll add either a commit or mustWork just after the paren parse and before the check for valid symbols and see if that doesn’t break any other code; shouldn’t based on how specialized that specific non namespaced operator parsing function is.

Reply:
I set the default number of alternative errors to print out to 1 and introduced the `--alt-error-count <nat>` option.